optimum/exporters/ipex/__init__.py

```diff
@@ -0,0 +1 @@
+from .model_patcher import export_model
```

Comment:
would remove it as we currently don't want user to use at the moment as it will modify the model inplace which can be risky for an uninformed user)

```suggestion
```

Reply:
Sure